CHANGES.md

```diff
@@ -28,6 +28,8 @@ Please follow the [`CHANGES.md` modification guidelines](https://github.com/dcos
 
 * Prune VIPs with no backends in order to avoid unbounded growth of state and messages exchanged among `dcos-net` processes. (DCOS_OSS-5356)
 
+* Fix support for big sets in the ipset mgr (COPS-5229)
+
```

Comment:
Thank youuu for adding an entry. Want to refer to this comment...: https://github.com/dcos/dcos/pull/6065/files#r313562300